VideoApp/VideoApp/Stores/TwilioAccessToken/CommunityTwilioAccessTokenStore.swift

```diff
@@ -35,9 +35,25 @@ class CommunityTwilioAccessTokenStore: TwilioAccessTokenStoreReading {
         let request = CommunityCreateTwilioAccessTokenRequest(
             passcode: authStore.passcode ?? "",
             userIdentity: appSettingsStore.userIdentity,
+            createRoom: true,
             roomName: roomName
         )
         
-        api.request(request) { completion($0.map { $0.token }) }
+        api.request(request) { [weak self] result in
+            guard let self = self else { return }
+            
+            if let roomType = try? result.get().roomType, roomType != self.appSettingsStore.remoteRoomType {
+                switch roomType {
+                case .group, .groupSmall, .unknown:
```

Comment:
We also set video codec in ApSettingsStore based on App Target... The difference is because...?

Reply:
In AppSettingsStore we are setting default value for video codec and that is different for community and internal builds. So at app install or sign out video codec would be set to default. 

In CommunityTwilioAccessTokenStore we're changing the setting (like a user could but we are instead automating it to help users out) depending on server config. So more like automatically changing setting here as opposed to just setting default value.

Reply:
That's kind of what I was thinking. Perfect. 🚢